manual_process_work/vbuild/Vtop__Syms.h

```diff
@@ -0,0 +1,57 @@
+// Verilated -*- C++ -*-
+// DESCRIPTION: Verilator output: Symbol table internal header
+//
+// Internal details; most calling programs do not need this header,
+// unless using verilator public meta comments.
+
+#ifndef _VTOP__SYMS_H_
+#define _VTOP__SYMS_H_  // guard
+
+#include "verilated_heavy.h"
+
+// INCLUDE MODULE CLASSES
+#include "Vtop.h"
+
+class Event
+{
```

Comment:
Is this needed - Verilator already supports "event" to some extent and I think it works...